### DISCUSSION TITLE
Application Rejected Multiple Times

### DISCUSSION THREAD

**Discussion by M3rcena** [community/community#53602] [GitHub Education] [answered]
Hello, 

I have submitted an application 3 times, the first time was because the document wasn't English so I got it officially translated with a qr code of the government which can be scanned and verified that the translation is valid. Then the second time it got rejected because the photo of the translated document was from my pc so I took a photo from my phone as it was saying and submitted it. Now the third time it got rejected again for the same reason: The photo should be from phone so it can't be manipulated. Well, it was from phone and I don't know why it got rejected and I was waiting three days now for a rejection of a reason that wasn't valid.

**Comment by samus-aran** [comment]
Hi @M3rcena 

Interesting you are saying that you are using a phone to take the image. Are you taking a live photo or picking an already taken photo from your phone's gallery when submitting?

**Reply by M3rcena** [reply]
I take a photo with my phone, put it in my computer and upload it in the application. Unfortunately the Google version on Android it doesn't work for me and I can't login while for one reason it's a loop for me so I only use pc. I applied again and this time I have the watermark on the photo and the timestamp which actually shows that it's a phone and it was taken today. Also if they scan the QR code on the translated document they can verify it by the greek government site that it's valid.

**Reply by M3rcena** [reply]
@samus-aran  if you got any suggestion that can help please let me know.

**Reply by M3rcena** [reply]
Hi, @samus-aran 

I just got emailed again that they declined it again with the same reason that I need to use a device camera. Well the photo they just rejected was taken from Xiaomi Redmi 9 and uploaded to the application from my pc. The photo was clearing saying in bottom left the phone and bottom right the day and time it was taken. I don't understand why they keep declining when I have done anything and I have submitted it so many times with a mobile photo.

**Reply by samus-aran** [reply, marked as answer]
Hi @M3rcena 

We really appreciate your patience on applying and I suspect if we resolve this last rejection reason we will be almost there. 

Now you've explained how you are providing the image we need to be explicit and ask you to take a live photo. Taking a photo via a camera and transferring it to a PC still counts towards being an uploaded file. I understand that this may be an inconvenience but if you are able to acquire a functional camera that you can take a live photo during the application process this would resolve your current rejection reason. You say you can't use Chrome on your android, try downloading another mobile browser like Firefox and see if you can apply on your phone that way.

**Reply by M3rcena** [reply]
Hi @samus-aran ,

Thank you for responding. I tried using the live photo from all the browser but when I click the button to use the front camera it doesn't change it. It only uses the front camera which makes it impossible to take a photo. Do we know anything about this problem because if I can't change the camera to the back then I can't actually use the live photo at all. Thank you for your help.

**Reply by M3rcena** [reply]
Hi @samus-aran ,

Quick Update. I managed and apply with a live photo from a different mobile where I could switch to the back camera. I am waiting for my response. It was saying it will take less than a day. I will let you know about the decision. Thank you!

**Reply by M3rcena** [reply]
Hello @samus-aran ,


I have to thank you for the last time. I managed to get approved after various attempts and your really helped me. Thank you a lot.

**Reply by samus-aran** [reply]
That's awesome @M3rcena  ! 🎉 Thanks so much for coming back and sharing the update. This will help other community members seeking similar answers. Start enjoying GitHub Global Campus 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋

**Reply by snasif** [reply]
I'm having the same issues, I don't know how to find help. My reason for being rejected are:

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I'm an online student and I've updated my profile mulitple times. Not sure what else to do.

**Comment by bornfamousTobia** [comment]
пн, 24 апр. 2023 г., 4:10 PM Michail Farmakis ***@***.***>:
